include/util/JsonRoot.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <Arduino.h>
+
+#include <cJSON.h>
+
+#include <string>
+#include <cstdint>
+
+namespace OpenShock {
+  class JsonRoot {
```

Comment:
What is this?

Reply:
Wrapper around cJSON to make it scoped, automatically de-allocated on scope exit

Reply:
ahhh your motivation was having the destructor, ic ic.